### PR TITLE
Fix focus issue when update distributed tracing for multiple devices

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: src/distributedTracingManager.ts [hunk 1]
```diff
@@ -43,7 +43,6 @@ export class DistributedTracingManager extends BaseExplorer {
             return;
         }
 
-        this._outputChannel.show();
         await this.updateDistributedTracingSettingForDevices(deviceIds, iotHubConnectionString, updateType, node);
```
Comment (formulahendry) [inline src/distributedTracingManager.ts]: What if deviceIds.length === 1?

File: src/distributedTracingManager.ts [hunk 2]
```diff
@@ -101,6 +100,7 @@ export class DistributedTracingManager extends BaseExplorer {
             title: `Update Distributed Tracing Setting`,
             location: vscode.ProgressLocation.Notification,
         }, async () => {
+            this._outputChannel.show();
             try {
                 const result = await this.updateDeviceTwin(mode, samplingRate, iotHubConnectionString, deviceIds);
```
Comment (formulahendry) [inline src/distributedTracingManager.ts]: updateDeviceTwin may takes long time. We only want to outputChannel.show() just before this.outputLine

Comment (SLdragon) [reply src/distributedTracingManager.ts]: Yes, that's right, however, if we put outputChannel.show() before outputLine, then we need to added it separately in try and catch block

File: src/distributedTracingManager.ts (hunk 1 above)
Comment (SLdragon) [reply src/distributedTracingManager.ts]: If devices.lenght ===1 then we will not use jobs

**Pull request merged**